tests/h/views/admin_nipsa_test.py

```diff
@@ -41,7 +41,7 @@ def test_add_flags_user(self, nipsa_service, pyramid_request, users):
 
         assert users['carl'] in nipsa_service.flagged
 
-    @pytest.mark.parametrize('user', ['', 'donkeys', '\x00'])
+    @pytest.mark.parametrize('user', ['', 'donkeys'])
```

Comment:
Given that the behaviour is now that this will raise an exception in `psycopg2` ([source](https://github.com/psycopg/psycopg2/issues/420)), rather than raise an exception in our code, I’m happy with this test disappearing.